packages/schema/src/language-server/validator/schema-validator.ts

```diff
@@ -20,10 +35,8 @@ export default class SchemaValidator implements AstValidator<Model> {
     }
 
     private validateDataSources(model: Model, accept: ValidationAcceptor) {
-        const dataSources = model.declarations.filter((d) => isDataSource(d));
-        if (dataSources.length === 0) {
-            accept('error', 'Model must define a datasource', { node: model });
-        } else if (dataSources.length > 1) {
```

Comment:
Is this check moved elsewhere?

Reply:
Added the check in the `LoadDocument` of CLI after imported modules merged.